Chandra/cmd_states/update_cmd_states.py

```diff
@@ -395,15 +395,19 @@ def main():
 
     # Set val to indicate if the output h5file is the "flight" version.
     # In that case use ftp directory cmd_states, else cmd_states_test.
-    is_flight = opt.h5file == '/proj/sot/ska/data/cmd_states/cmd_states.h5'
+    is_flight = ((opt.h5file == '/proj/sot/ska/data/cmd_states/cmd_states.h5')
+                 or (opt.server == '/proj/sot/ska/data/cmd_states/cmd_states.db3'))
     ftp_dirname = 'cmd_states' if is_flight else 'cmd_states_test'
 
     # If running on the OCC (GRETA) network then just try to get a new HDF5
     # file from lucky in /home/taldcroft/cmd_states and copy to opt.h5file.  The
     # file will appear on lucky only when the HEAD network version gets updated
     # with changed content.
-    if opt.occ and opt.h5file:
-        occweb.ftp_get_from_lucky(ftp_dirname, [opt.h5file], logger=logging)
+    if opt.occ:
+        if opt.server:
+            occweb.ftp_get_from_lucky(ftp_dirname, [opt.server], logger=logging)
```

Comment:
The sqlite database is about 200Mb (compared to the h5 file which is more like 20).  I'm not sure about a legitimate concern that a problem with pushing could fill up user quota on lucky.